modules/cnf-du-updating-performance-addon-operator.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-updating-performance-addon-operator.adoc_{context}"]
+
+= Updating Performance Addon Operator
+
+After you have configured workload partitioning, you need to ensure that the Performance Addon Operator has been installed and configured on a
```

Comment:
nit: "on a Performance Profile" reads awkwardly. Maybe "...installed and that you configured a performance profile"?

Please use either "performance profile" or "`PerformanceProfile` object" according to the [API object formatting](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting) section of the doc guidelines.

Regarding the link, don't.

Unless you have some special circumstance, the commonly used workaround is to edit the file for the assembly like so:

```
include::modules/cnf-du-updating-performance-addon-operator.adoc[leveloffset=+1]

.Additional resources

* xref:../../i-dont-know-the-path/cnf-performance-addon-operator-for-low-latency-nodes.adoc[Performance Addon Operator for low latency nodes]
```